.github/workflows/pullrequest.yml

```
@@ -639,13 +639,10 @@ jobs:
       - name: Check prepare success
         run: '[[ ${{ needs.prepare.result }} == "success" ]] || exit 1'
       - name: Check if deploy-feature is enabled
+        if: ${{ needs.prepare.outputs.DEPLOY_FEATURE }}" == 'true'
```

actionlint (1.7.4)

if: condition "${{ needs.prepare.outputs.DEPLOY_FEATURE }}\" == 'true'" is always evaluated to true because extra characters are around ${{ }}

(if-cond)